Packs/CortexXpanse/ReleaseNotes/1_0_4.md

```diff
@@ -0,0 +1,72 @@
+
```

Comment:
Please make sure the release notes (especially prefixes) match the template: https://xsoar.pan.dev/docs/documentation/release-notes#general